### PR TITLE
fix cloud reflection and update manifest.json

### PR DESCRIPTION
**Pull request by bambosan**

### CODE REVIEW

**Comment by jebbyk** [comment]
yeee, gona check it  (◠﹏◠)

**Review by jebbyk — CHANGES_REQUESTED** [review]
I've checked it. Seems like water needs some tweaking (maybe only normal map). I'll show a screenshot a bit later.

**Review by jebbyk — COMMENTED** [review]
Screenshot_20210407-200433

**Comment by jebbyk** [comment]
Screenshot_20210407-200934
Screenshot_20210407-200916
horizon color at night should be tuned too

**Comment by bambosan** [comment]
yes water problem is normalmap and I'll correct the horizon color

**Comment by jebbyk** [comment]
8x reduced normals strength resulted in this. clouds reflection is much more distorted then the highlight. It is strange. I think there is an issue with vectors calculation somewhere probably even not in the reflection code but in mapping or in specular and diffuse lighting calculations idk.
Screenshot_20210408-103821

**Comment by bambosan** [comment]
yep,,  because of this

**Comment by bambosan** [comment]
If the reflectedVector is replaced with a normalized relative position it
will be fine and there is no need to reduce the strength of the normalmap

**Comment by bambosan** [comment]
pos for cloud reflection using reflectedVector which post has normalmap
data and for cloud mounting also use result of dot production from
reflectedVector
well it will make it like in the picture

**Comment by bambosan** [comment]
sorry I just realized it now

**Comment by bambosan** [comment]
sorry I just realized it now

**Comment by jebbyk** [comment]
email replies a bit confusing :-D

**Comment by OpenSauce04** [comment]
@Mcbamboo Make sure to check how this looks with the new water normals

File: shaders/glsl/includes/reflections.glsl [hunk 1]
```diff
@@ -2,37 +2,41 @@
...
 	vec3 zenithColor = vec3(0.25, 0.5, 1.0) * length(resultLighting);
-	return vec4(mix(zenithColor, horizonColor, horizonLine), horizonLine);
+	return vec4(mix(zenithColor, horizonColor, horizonLine), isZenithSky);
 }
 
-vec3 buildSkyPlaneReflection(vec3 relativePosition, vec3 normalColor, vec4 skyLightReflected, float isRain, float roughness){
+vec3 buildSkyPlaneReflection(vec3 reflectedVector, vec4 skyLightReflected, float isRain, float roughness){
+
+	highp vec3 cldCoord = reflectedVector;
+		cldCoord /= cldCoord.y;
```
Comment (jebbyk) [inline shaders/glsl/includes/reflections.glsl:21]: I think we can get rid of this cuz probably reflect() already do this job. And also it is probably causing water reflection issue

**Comment by jebbyk** [comment]
Ive checked some real photos of the water and I think it is everything ok with shader. And I have to reduce normal map strength

**Comment by bambosan** [comment]
will it work well? sorry I just replied now

**Pull request closed without merging**